appengine-vmruntime/vmruntime/middleware.py

```diff
@@ -37,7 +37,7 @@
     'USER_IS_ADMIN': '0',  # Default admin flag to explicit '0'.
     'USER_NICKNAME': '',
     'USER_ORGANIZATION': '',
-    'SERVER_SOFTWARE': 'Google App Engine/0.62',
+    'SERVER_SOFTWARE': 'Google App Engine/0.64',
```

Comment:
Will you file a bug and assign it to me to fix this so that it pulls from the distribution?